nvdaHelper/local/wasapi.cpp

```diff
@@ -351,9 +351,11 @@ HRESULT WasapiPlayer::feed(unsigned char* data, unsigned int size,
 				}
 			}
 			hr = client->GetCurrentPadding(&paddingFrames);
-			if (hr == AUDCLNT_E_DEVICE_INVALIDATED) {
-				// If we're using a specific device, it's just been invalidated. Fall back
-				// to the default device.
+			if (hr == AUDCLNT_E_DEVICE_INVALIDATED ||
+					hr == AUDCLNT_E_NOT_INITIALIZED) {
```

Comment:
Lint for readability

```suggestion
			if (
				hr == AUDCLNT_E_DEVICE_INVALIDATED
				|| hr == AUDCLNT_E_NOT_INITIALIZED
			) {
```